gradle/plugins/src/main/kotlin/com.hedera.gradle.platform-publish.gradle.kts

```diff
@@ -88,8 +88,19 @@ publishing.repositories {
 // Register one 'release*' task for each publishing repository
 publishing.repositories.all {
     val ucName = name.replaceFirstChar { it.titlecase() }
-    tasks.register("release$ucName") {
-        group = "release"
-        dependsOn(tasks.named("publishMavenPublicationTo${ucName}Repository"))
+    val taskName = "release$ucName"
+
+    // Register the task if it is not already registered
```

Comment:
Not actually sure this is needed, we may just need the conditional below to modify the existing empty task that was defined at the root, in the lifecycle file. 

I was getting the error: `Cannot add task 'releasePrereleaseChannel' as a task with that name already exists.` without these conditionals because of the empty task definitions in the lifecycle file. Do we have two execution paths here?